docs/meshstack.azure.index.md

```diff
@@ -395,3 +402,112 @@ The arguments available here are:
+- `Group.ReadWrite.All` - this permissions is required to create new groups
+- `User.Invite.All` - this permission is required if you want to enable B2B User Invitations
+- `User Access Administrator` - to add and remove user group access to the Subscriptions
+- `Rights to call Azure Functions` - see the [end of this section](#azure-function-access-setup) for how to setup this
+
+Created Subscriptions will have the Service Principal of the replicator registered as owner at first. As soon as all needed maintainance steps are performed (like renaming the subscription, moving it to the final management group etc.), the replicator removes itself as an owner.
+
+All permissions left are therefore granted only via the management group hierarchy. The meshstack software does **not** need access related to actual workload inside this subscriptions, however to perform certain maintenance tasks the following permissions/roles must be granted to the replicator principal through the management group hierarchy:
+
+- `Blueprint Operator` - required for assigning and managing Blueprint assignments to the Subscriptions
+- `Management Group Contributor` - required on the target management group for moving Subscriptions
+
+In order to prevent that the replicator is able to assign himself more permissions than originally anticipated its recommended to add a policy rule like this:
+
+```json
+{
+    "properties": {
+        "mode": "all",
+        "displayName": "meshcloud Privilege Escalation Prevention",
+        "description": "Prevent meshcloud SP from assigning itself new roles.",
+        "policyRule": {
+          "if": {
+            "allOf": [
+              {
+                "equals": "Microsoft.Authorization/roleAssignments",
+                "field": "type"
+              },
+              {
+                "allOf": [
+                  {
+                    "field": "Microsoft.Authorization/roleAssignments/roleDefinitionId",
+                    "equals": "/subscriptions/<SUBSCRIPTION_ID>/providers/Microsoft.Authorization/roleDefinitions/<AZURE_SERVICE_PRINCIPAL_ROLE_ID>"
+                  },
+                  {
+                    "field": "Microsoft.Authorization/roleAssignments/principalId",
+                    "equals": "<AZURE_SERVICE_PRINCIPAL_ID>"
+                  },
+                ]
+              }
+            ]
+          },
+          "then": {
+            "effect": "audit"
+          }
+      }
+    }
+}
+```
+
+If you intend to call Azure functions via a Landing Zone configuration then you need to grant the replicator principal also permissions to the role you have created for the Azure secured function call, described [here](#secure-azure-function-setup).
+
+#### Azure Function Access Setup
+
+In order to make an Azure Function only accessible via the replicators Service Principal follow these steps:
+
+1. Create a system assigned identity for your function.
```

Comment:
This step is not required for the function access. This only allows your function to have permissions inside Azure (like starting VMs, Connecting LogWorkspaces and such)